test/README.md

```diff
@@ -12,9 +14,219 @@ To run all unit tests:
 go test ./...
 ```
 
+## Test library
+
+You can use the test library in this dir to:
+
+* [Use common test flags](#use-common-test-flags)
+* [Output logs](#output-logs)
+* [Emit metrics](#emit-metrics)
+* [Ensure test cleanup](#ensure-test-cleanup)
+
+### Use common test flags
+
+These flags are useful for running against an existing cluster, making use of your existing
+[environment setup](https://github.com/knative/serving/blob/master/DEVELOPMENT.md#environment-setup).
+
+By importing `github.com/knative/serving/test` you get access to a global variable called
+`test.Flags` which holds the values of [the command line flags](/test/README.md#flags).
+
+```go
+imagePath := strings.Join([]string{test.ServingFlags.DockerRepo, image}, "/"))
```

Comment:
add some other flag example. This flag is for serving repo

Reply:
good call!